# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1220901704 14400
#      Mon Sep 08 15:21:44 2008 -0400
# Node ID 0ec09255515a761aa2b4c7b57cda3bea23c7355d
# Parent  3eda9aa17d7c40721d92ab385bf26acce980720c
add colorbar docstring to the manual

diff --git a/doc/interpreter/plot.txi b/doc/interpreter/plot.txi
--- a/doc/interpreter/plot.txi
+++ b/doc/interpreter/plot.txi
@@ -337,16 +337,18 @@ border lines are on.
 @DOCSTRING(text)
 
 @DOCSTRING(xlabel)
 
 @DOCSTRING(box)
 
 @DOCSTRING(grid)
 
+@DOCSTRING(colorbar)
+
 @node Multiple Plots on One Page
 @subsection Multiple Plots on One Page
 
 Octave can display more than one plot in a single figure.  The simplest
 way to do this is to use the @code{subplot} function to divide the plot
 area into a series of subplot windows that are indexed by an integer.
 For example,
 
